<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/Servicios.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_puntos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>751</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>con servicio: Monte Alto - Azteca.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_no_apagar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>751</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>No apague la boletera, datos a√∫n no enviados del viaje</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_datos_faltantes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>751</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 22pt &quot;Arial Black&quot;;
color: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>Faltan 6 datos por enviar.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
